p/scripts/user.query.js

```diff
@@ -11,27 +11,63 @@ function init_draggable() {
 		return;
 	}
 	
-	let source;
+	var source;
 	const configureQueries = document.querySelector('#configureQueries');
+	const addMarker = function(position, element) {
+		const hr = configureQueries.querySelector('hr.drag-drop-marker');
+		if (null === hr) {
+			element.insertAdjacentHTML(position, '<hr class="drag-drop-marker"/>');
```

Comment:
```suggestion
			element.insertAdjacentHTML(position, '<hr class="drag-drop-marker" />');
```
Minor consistency :-)